upup/pkg/fi/cloudup/scalewaytasks/loadbalancer.go

```diff
@@ -220,10 +219,11 @@ func (_ *LoadBalancer) RenderTerraform(t *terraform.TerraformTarget, actual, exp
 	}
 
 	tfLB := terraformLoadBalancer{
-		Type: "LB-S",
-		Name: expected.Name,
-		Tags: expected.Tags,
-		IPID: terraformWriter.LiteralProperty("scaleway_lb_ip", tfName, "id"),
+		Type:        "LB-S",
```

Comment:
Could it be a default const that could be changed easily in case of a rename or change of product?